draft-regext-rdap-x-media-type.md

```diff
@@ -189,11 +189,36 @@ Provisional Registration: No
 
 # Acknowledgements
 
-Pawel Kowalik provided ideas and feedbacks that have contributed to
+Pawel Kowalik and James Mitchel have provided ideas and feedbacks that have contributed to
```

Comment:
IIRC, Mario also helped start the `vary` header discussion.

Reply:
He was saying that clients should send the vary header, nor did he ever offer a reason to use the vary header but rather his intent was to poke holes. James Mitchell actually had a real conversation with me about this, though it was a private conversation. I don't view Mario's comments as contributions because there was nothing of substance in them.